ccm_web/client/src/components/UserEnrollmentForm.tsx

```diff
@@ -178,7 +178,7 @@ export default function UserEnrollmentForm (props: UserEnrollmentFormProps): JSX
         Enter the user&apos;s non-UM email address, and click &quot;Search&quot; to see if they are in Canvas. (Required)
       </Typography>
       {
-        showIncompleteAlerts && userExists === undefined &&
+        showIncompleteAlerts && (userInfo === undefined) &&
```

Comment:
Just FYI, parentheses aren't really necessary here, but not a big deal, they can help readability.

Reply:
Fine to leave this as is.